changelog/unreleased/issue-3183

```diff
@@ -0,0 +1,6 @@
+Bugfix: Allow specify --tag option several times and the resulting backup must contain all the specified tags.
```

Comment:
Suggestion: "Bugfix: Allow specify --tag option several times for backup command"